lib/pdf/elements/list.rb

```diff
@@ -5,12 +5,17 @@ def self.render(pdf, element, indent: 0)
     index = 0
     pdf.indent indent do
       element.css("> li").each do |li|
-        inner_list = li.children.css("ul,ol")
+        text = li.children
```

Comment:
I see, but I guess it shouldn't be a problem, since we are printing markdown, the `ul` or `ol` will be the last elements on the `li`